### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by mohammad-al-zoubi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,125 +1,80 @@
...
-`flow start UpdateRegulation name: Regulation2, description: Some Description, version: 0.2, releaseDate: 2022-03-22, linearId: <RegulationLinearId>`
-
-## Create Rule Flow 
-`flow start CreateRule name: NewRule, ruleSpecification: new Description, parentRegulationLinearId: <RegulationLinearId>`
-
-`run vaultQuery contractStateType: com.compliance.states.Rule`
+- The FSP claims its compliance to the regulations and has to prove his claims to the auditor
 
-## Update Rule Flow
+- If the auditor is not convinced by the proof provided by the FSP or considers the proven claims not sufficient to prove compliance, he can request examining terabytes of logs from all IT systems of the FSP. This process is long and very demanding.
```
Comment (viganll) [inline README.md:14]: he is not gender neutral

Comment (alzoubi36) [reply README.md:14]: I am gynophobic :D

Comment (alzoubi36) [reply README.md:14]: Can you suggest another pronoun?

**Comment by viganll** [comment]
There is no indication at the beginning of the read me file that the user can run the modules separately. We should make it clear that in case they are interested to run the cordapp only once they can follow the readme that Valentin wrote.

**Pull request merged**